apps/earn-protocol-landing-page/components/layout/MasterPage/MasterPage.tsx

```
@@ -1,24 +1,16 @@
 import { type FC, type PropsWithChildren } from 'react'
-import { parseServerResponseToClient } from '@summerfi/app-utils'
-
-import systemConfigHandler from '@/server-handlers/system-config'
 
 import masterPageStyles from './MasterPage.module.scss'
 
 interface MasterPageProps {}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)